modules/monitoring-configuring-pod-topology-spread-constraints.adoc

```diff
@@ -0,0 +1,189 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-pod-topology-spread-constraints_{context}"]
+= Configuring pod topology spread constraints
+
+You can configure pod topology spread constraints for 
+ifndef::openshift-dedicated,openshift-rosa[]
+the pods deployed by the Cluster Monitoring Operator
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+the pods for user-defined monitoring
+endif::openshift-dedicated,openshift-rosa[]
+to control how pod replicas are scheduled to nodes across zones.
+This ensures that the pods are highly available and run more efficiently, because workloads are spread across nodes in different data centers or hierarchical infrastructure zones.
+
+You can configure pod topology spread constraints for monitoring pods by using 
+ifndef::openshift-dedicated,openshift-rosa[]
+the `cluster-monitoring-config` or 
+endif::openshift-dedicated,openshift-rosa[]
+the `user-workload-monitoring-config` config map.
+
+.Prerequisites
+
+ifndef::openshift-dedicated,openshift-rosa[]
+* *If you are configuring pods for core {product-title} monitoring:*
+** You have access to the cluster as a user with the `cluster-admin` cluster role.
+** You have created the `cluster-monitoring-config` `ConfigMap` object.
+* *If you are configuring pods for user-defined monitoring:*
+** A cluster administrator has enabled monitoring for user-defined projects.
+** You have access to the cluster as a user with the `cluster-admin` cluster role, or as a user with the `user-workload-monitoring-config-edit` role in the `openshift-user-workload-monitoring` project.
+** You have created the `user-workload-monitoring-config` `ConfigMap` object.
```

Comment:
since 4.11, the `user-workload-monitoring-config` ConfigMap is created by default, we can remove this line

Reply:
I missed that, thanks!

Reply:
Oh thank you for pointing this out @juzhao ! I actually cannot find that information anywhere in our docs and this is a prerequisite in a lot of our procedures. I think this should therefore get fixed in our docs. I can create a Jira for it and revise our docs then, deleting the line where needed and maybe replacing it with `A cluster administrator has enabled monitoring for user-defined projects.` if that is not in the prereqs (e.g. [here](https://docs.openshift.com/container-platform/4.15/observability/monitoring/configuring-the-monitoring-stack.html#moving-monitoring-components-to-different-nodes_configuring-the-monitoring-stack))
WDYT?

Reply:
@eromanova97 sounds good, I think remove `You have created the `user-workload-monitoring-config` `ConfigMap` object` is fine, since there is usually `A cluster administrator has enabled monitoring for user-defined projects` along with it in our docs.
FYI: the change is brought by 4.11 PR: https://github.com/openshift/cluster-monitoring-operator/pull/1603

Reply:
Yes, thank you once again for the information!